src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery newGame = new Lottery();
+        System.out.println(newGame.getRandomBall().toString());
+        System.out.println(newGame.getRandomBall().toString());
+        System.out.println(newGame.getRandomBall().toString());
```

Comment:
let's use a loop for this purpose. And we don't need to call `toString()` method in `System.out.println`, it will be called "under the hood"